.gitignore

```diff
@@ -9,7 +9,7 @@ npm-debug.log
 .eslintcache
 package-lock.json
 /tools/set-import/sets
-databases/sqlite.db
+databases/*.db
```

Comment:
This needs to be `databases/*.db*` as I said earlier. If you enable [WAL](https://www.sqlite.org/wal.html) (which PS does for certain databases with high write traffic) you get `.db.shm` and `.db.wal` files as well.